octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/search/SearchEditor.java

```diff
@@ -50,7 +50,8 @@
     // private static final ILog logger = Activator.getDefault().getLog();
 
     public static final String ID = "com.hpe.octane.ideplugins.eclipse.ui.search.SearchEditor";
-
+    private String darkBackgroundColorString = "rgb(52,57,61)"; 

```

Comment:
this looks very specific? Isn't there any way to retrieve it from the Eclipse themes?